tools/dds/dds-server/dds-device-broadcaster.h

```diff
@@ -112,5 +109,6 @@ namespace tools
         active_object<> _new_client_handler;
         std::condition_variable _new_client_cv;
         std::mutex _new_client_mutex;
-    };  // class dds_server
+        bool init_done;
```

Comment:
Not `_init_done`? Do we really need it? Maybe we can use the `_running` shared_ptr which will be initialized in `init()`?

Reply:
Changed as discussed